common/amundsen_common/models/lineage.py

```diff
@@ -16,6 +16,7 @@ class LineageItem:
     source: str  # database this resource is from
     badges: Optional[List[Badge]] = None
     usage: Optional[int] = None  # statistic to sort lineage items by
+    parent: Optional[str] = ''  # key of the parent entity, used to create the relationships in graph
```

Comment:
why default to empty string instead of None, given it's optional?

Reply:
fixed!